checker/jdk/nullness/src/java/util/concurrent/ConcurrentHashMap.java

```diff
@@ -0,0 +1,42 @@
+package java.util.concurrent;

```

Comment:
Why do you use two very different styles for the two classes? For this first one you just have the stubs and for the other one the complete source code.